### DISCUSSION TITLE
Spread operator in config causes syntax error

### DISCUSSION THREAD

**Discussion by umarsheikh13** [tailwindlabs/tailwindcss#2658] [Help]
Hey everyone

I'm trying to use tailwindcss (v1.9.5) with webpack but every time I run the dev server I receive the following error:

```
    SyntaxError: /app/public/tailwind.config.js: Unexpected token (90:12)
      88 |         },
      89 |         borderColor: theme => ({
    > 90 |             ...theme('colors'),
         |             ^
      91 |             default: theme('colors.gray.300', 'currentColor'),
      92 |         }),
```

The version of node I'm using is 12.3.1 and I've tried everything but no luck.

**Comment by mubaraqwahab** [comment]
Hello. Could it be related to this? https://stackoverflow.com/q/50327580/12695621
